parachain-tracer/src/parachain_block_info.rs

```diff
@@ -115,51 +108,10 @@ enum ParachainBlockState {
 mod tests {
 	use crate::test_utils::create_para_block_info;
 
-	#[test]
```

Comment:
Tests need to be updated, not removed.

Reply:
The method it tested doesn't exist anymore.